### ISSUE TITLE
Dead Mine dungeon

### ISSUE THREAD

**Issue by cNicky** [Athissa/Tracker#400]
**Describe the issue you're having**:
On second boss, we weren't able to attack it. We died to try to reset it but we weren't able to revive, tough we were all dead, we got the message someone was still fighting. We disbanded group to leave dungeon and me and 1 of my friends end up in a diferent area of the one we were, and there was no graveyard spirit to revive us. We got unstuck throw the site.
**Explain how you expect it work**:

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Links to Wowhead, YouTube, etc**:

**Comment by Ygdrassill** [comment]
Yes  the 2nd boss is bugd   iwen lvl 110 having trubel killing it, u can only atack it when he jumps on somone but  masive dmg needed cuz it regens  hp,so low lvls  canot realy  do it
